maquetado/scss/detail.scss

```diff
@@ -1,31 +1,75 @@
 @import "colors";
 
 .book-title {
+  font-family: Roboto;
   font-size: 30px;
   font-weight: bold;
   margin-right: 10px;
+  line-height: 40px;
 }
 
 .book-detail-title {
   color: $black;
+  font-family: Roboto;
   font-size: 20px;
   font-weight: bold;
+  line-height: 25px;
   margin-right: 10px;
 }
 
 .book-description {
   color: $grey;
+  font-family: Roboto;
   font-size: 20px;
+  line-height: 25px;
 }
 
 .book-genre {
   color: $grey;
-  font-size: 25px;
   font-weight: bold;
+  font-size: 30px;
+  line-height: 40px;
+}
+
+.line-separator {
+  position: relative;
+  &::after {
+    background-color: $green;
+    content: "";
+    height: 5px;
+    margin-top: 20px;
+    position: absolute;
+    width: 640px;
+    top: calc(70%);
```

Comment:
I thought you would have to use more than 100%. If it less then it is not worth to use percentage, 

I'll explain myself 

In some cases you will have to set `bottom: -sarasa(px)`. It's not a good idea to use negative values, for those cases we can use `top: calc(100% + some pixels)`

Since this is not the case then it's not worth to use calc or percentages, instead we can use pixels which is more predictable and ease to read.